src/utils/PaymentsConverter.ts

```diff
@@ -145,7 +146,20 @@ export function getNodoAttivaRPTInput(
       codiceIdRPT,
       datiPagamentoPSP: {
         importoSingoloVersamento:
-          paymentActivationsPostRequest.importoSingoloVersamento / 100
+          paymentActivationsPostRequest.importoSingoloVersamento / 100,
+        soggettoPagatore: paymentActivationsPostRequest.soggettoPagatore
```

Comment:
for clarity it would be better to move this logic above and use an intermediate variable (i.e. `soggettoPagatore`)

Reply:
Fixed in 0b9d92d